packages/js/github-actions/utils/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
+}
```

Comment:
:grey_question: 
Why is that needed? Isn't the `packages/js/github-actions/package.json`'s `.type: "module"` enough?

Reply:
Thanks for catching this. I didn't set package.json in the package root directory when experimenting in another repo and forgot here already has. Removed in https://github.com/woocommerce/grow/pull/22/commits/89644a30c92e3797fcfa4bb08ae319553e344592.